src/Hal/Serializer/CollectionNormalizer.php

```diff
@@ -57,6 +57,9 @@ public function supportsNormalization($data, $format = null)
     public function normalize($object, $format = null, array $context = [])
     {
         $data = [];
+        $currentPage = 1;
+        $lastPage = 1;
+        $itemsPerPage = 0;
```

Comment:
Those where added by @Simperfit. I'll revert them considering [this](https://github.com/api-platform/core/pull/785#discussion_r90441569).